### PR TITLE
Feature/cml/layout component

### PR DESCRIPTION
**Pull request by Astro-Alien**

### CODE REVIEW

File: components/layout-container/layout-container.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+export default class LayoutContainer extends crsbinding.classes.BindableElement {
```
Comment (Ashton-Pragma) [inline components/layout-container/layout-container.js]: I dont think this needs to be a bindable element its overkill

File: components/layout-container/layout-container.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<style>
+    * {
+        box-sizing: border-box;
```
Comment (Ashton-Pragma) [inline components/layout-container/layout-container.html]: Are we expecting border styling on this?

**Review by Ashton-Pragma — CHANGES_REQUESTED** [review]

File: components/layout-container/layout-container.js (hunk 1 above)
Comment (Astro-Alien) [reply components/layout-container/layout-container.js]: I changed it to a HTMLElement.

File: components/layout-container/layout-container.html (hunk 2 above)
Comment (Astro-Alien) [reply components/layout-container/layout-container.html]: The .html of the component has now been removed as it is not required anymore.

**Review by Ashton-Pragma — APPROVED** [review]

**Pull request merged**